data/presets/presets/shop/supermarket.json

```diff
@@ -12,7 +12,11 @@
     "terms": [
         "grocery",
         "store",
-        "shop"
+        "shop",
+        "grocery store"
```

Comment:
The build failed because you're missing commas after these values, but I'll just add them for you.

Reply:
Thanks!